src/dynamics/time_integration.jl

```diff
@@ -10,30 +10,57 @@ Base.@kwdef struct Leapfrog{NF} <: TimeStepper{NF}
 
     # OPTIONS
     "time step in minutes for T31, scale linearly to `trunc`"
-    Δt_at_T31::Float64 = 30
+    Δt_at_T31::Second = Minute(30)
 
     "radius of sphere [m], used for scaling"
     radius::NF = 6.371e6
 
+    "adjust Δt_at_T31 to the output_dt, so that it is rounded to closest"
+    "divisor of output_dt, ensures that output_dt is kept exactly in the output"
+    adjust_Δt_with_output::Bool = false 
+
+    "Output time step [hrs], needed if adjust_Δt_with_output is true"
+    output_dt::Second = Hour(6)
+
     # NUMERICS
     "Robert (1966) time filter coefficeint to suppress comput. mode"
     robert_filter::NF = 0.05
 
     "Williams time filter (Amezcua 2011) coefficient for 3rd order acc"
     williams_filter::NF = 0.53
 
-
     # DERIVED FROM OPTIONS    
     "time step Δt [s] at specified resolution"
-    Δt_sec::Int = round(Int,60*Δt_at_T31*(32/(trunc+1)))
+    Δt_sec::Second = get_Δt_sec(Second(Δt_at_T31), trunc, adjust_Δt_with_output, Second(output_dt))
```

Comment:
What I meant with #417 is really just the user facing time-related parameters. One doesn't choose `Δt_sec` directly so this can actually be of type `NF` and therefore doesn't have to be rounded to the second which addresses my point in #416

Reply:
adressed with commits below